ledfx_docker/start.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+${PULSEAUDIOCLIENT:-"0"}
+set -eu
+
+if [ "$PULSEAUDIOCLIENT"!="0" ]; then
```

Shellcheck (0.10.0)

[error] You need spaces around the comparison operator.

(SC2077)